src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -71,9 +71,12 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
 
     DockerBuildImage() {
         ext.getImageId = { imageId }
-
+        onlyIf {
+            boolean runTask = previouslyBuiltImageExists() ? false : true
+            if (!runTask) { logger.quiet("${this.name}: UP-TO-DATE") }
```

Comment:
This looks like a hack to me. Why can't we use the regular UP-TO-DATE check anymore?